windows/src/developer/TIKE/main/UframeOnScreenKeyboardEditor.pas

```diff
@@ -311,6 +324,13 @@ procedure TframeOnScreenKeyboardEditor.tmrUpdateCharacterMapTimer(Sender: TObjec
 { - Tab interactions                                                         - }
 { ---------------------------------------------------------------------------- }
 
+procedure TframeOnScreenKeyboardEditor.pagesChange(Sender: TObject);
+begin
+  inherited;
+  if pages.ActivePage = pageCode then
+    frameSource.SetFocus;
```

Comment:
Just checking if this is intended to be identical to `TframeOnScreenKeyboardEditor.SetFocus`? I don't see where `Sender` is used.

Reply:
Good question. It is intended to be the same code, yes. There's not a lot of value in merging this code with `SetFocus` -- the `SetFocus` call also has a call to `inherited [SetFocus]`, and for the sake of one line of code, refactoring is unnecessary I think.

In this case, `Sender` is unused -- it's just the `pages` component. 

Actually, the `inherited` call here is extraneous as well (as the parent class does not have an instance of `pages`), so have removed it.